addons/mail/static/src/views/web/fields/emojis_char_field/emojis_char_field.xml

```diff
@@ -5,6 +5,10 @@
         <xpath expr="//span[1]" position="attributes">
             <attribute name="t-ref">targetReadonlyElement</attribute>
         </xpath>
+        <xpath expr="//*[hasclass('o_field_input_buttons')]" position="attributes">
+            <attribute name="class" add="emoji"  separator=" "/>           
+        </xpath>
```

Comment:
Isn't it possible to use the class `o_mail_emojis_buttons` inside the css rule instead of adding a class that isn't really descriptive. This template `mail.EmojisFieldButton` adds the button with the element that contains the class.
So your scss rules could be changed to 
```
.o_field_widget:has(.o_field_input_buttons > .o_mail_emojis_buttons)
.o_field_input_buttons:has(.o_mail_emojis_buttons)
```

Reply:
You're right it is working, I changed it.